developers/weaviate/starter-guides/compression.mdx

```diff
@@ -0,0 +1,149 @@
+
+- BQ uses fast, bitwise calculations. The flat index relies on brute-force search so it is calculation intensive. BQ's bitwise calculations are extremely efficient. Searches of BQ compressed data can be as much as [10 to 20 times as fast](/binary-quantization#-performance-improvements-with-bq) as similar searches of uncompressed data and with equivalent rates of recall. BQ is sensitive to the underlying data. If you use a flat index, evaluate BQ compression to verify the performance improvements with your data set.
+
+- SQ significantly improves search speeds. SQ is new in v1.26. It is faster than PQ, perhaps 3 to 4 times as fast as searching uncompressed data. SQ has a higher dimensional resolution than BQ that helps recall. Look for an upcoming blog post that discusses the tradeoffs with SQ compression.
+
+SQ and BQ both have optional vector caches. Use these configurable caches to load frequently used, uncompressed vectors into memory to improve overall search times.
+
+#### Import speed
+
+Importing and compressing vectors takes slightly longer than just importing them, but this is a one time cost. In contrast, loading a compressed index into memory is faster since there is less data to load.
+
+Starting in v1.22, Weaviate has an optional, [asynchronous indexing](/developers/weaviate/config-refs/schema/vector-index#asynchronous-indexing) feature which improves the import process. When asynchronous indexing is enabled, indexing takes place as a background process. You can begin other tasks without having to wait for the entire import to be indexed. This means you are able to begin other tasks sooner.
+
+### Activate compression
+
+PQ and SQ both require training data. PQ has to define centroids for each segment. SQ has to determine the minimum and maximum values for the bucket boundaries. When you have imported a large enough training set, the algorithm will compress your data.
+
+SQ has to be enabled when you create the collection.
+
+If you have async indexing and [AutoPQ enabled](/developers/weaviate/configuration/compression/pq-compression#configure-autopq), PQ compression can be started anytime. If not, you should only enable PQ after you have imported enough objects to [train the algorithm](/developers/weaviate/configuration/compression/pq-compression#manually-configure-pq).
+
+BQ doesn't require a training step, however it can only be enabled when you create your collection. BQ cannot be enabled after you start to add data to the collection.
+
+## Recommendations
+
+Most applications benefit from compression.
+
+- The cost savings are significant. In [Weaviate Cloud](https://weaviate.io/pricing), for example, uncompressed collections can be 5 to 8 times as expensive as compressed collections.
```

Comment:
Let's focus this sentence on the positive side and say that compressed collections can yield xyz savings! (not necessarily in these words)